README.md

```diff
@@ -34,6 +34,8 @@ Invoke `gulp` to rebuild your app and regenerate the script that offlines it. In
 gulp && gulp deploy
 ```
 
+*At least one commit to the repository is required for successful deploy.*
```

Comment:
Nice! We might want to suggest that the developer can commit the changes that the bootstrap command just made, and give them a sample command to do it, something like:

```bash
git add . && git commit -m"initial version of app"

But we can also wordsmith that in a followup.